python/paddle/v2/framework/tests/test_rmsprop_op.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+import numpy as np
+from op_test import OpTest
+
+
+class TestRmspropOp(OpTest):
+    def setUp(self):
+        self.op_type = "rmsprop"
+
+        param = np.random.random((123, 321)).astype("float32")
+        grad = np.random.random((123, 321)).astype("float32")
+        moment = np.zeros((123, 321)).astype("float32")
+        learning_rate = np.array([0.01]).astype("float32")
```

Comment:
why is 'learning_rate' not an attribute?

Reply:
As Abhinav commented above, I think I will retain this as an input for now, given that we are doing the same in all other PRs too.